train.py

```
@@ -497,7 +513,7 @@ def parse_opt(known=False):
     parser.add_argument('--balanced', action='store_true', help='use weightedrandomsampler for image selection in training')
     parser.add_argument('--local_rank', type=int, default=-1, help='Automatic DDP Multi-GPU argument, do not modify')
     parser.add_argument('--eval-batchfactor', type=int, default=2, help='factor to increase or decrease batch size during evaluation')
-
+    parser.add_argument('--eval-x-epochs', type=int, default=10, help='evaluate only every x epochs')
```

Can we raise an error when both  `noval` and `eval-x-epochs` are specified?